modules/standard/Curl.chpl

```diff
@@ -90,181 +430,355 @@ proc slist.append(str:string) {
   if err then ioerror(err, "in slist.append()");
 }
 
+/* Free an slist. Chapel programs must call this function after using an slist.
+   Programs must ensure that there are no ongoing connections using
+   this slist when it is freed. For an example, see :ref:`curl-example-7`.
+ */
 proc slist.free() {
   on this.home {
     chpl_curl_slist_free(this.list);
   }
 }
 
-// These are meant to be ued with the file.setopt() function. This way, a user has
-// access to the easy interface.
+// These are meant to be ued with the file.setopt() function. This way, a user 
+// has access to the easy interface.
+
+
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_WRITEDATA.html */
 extern const curlopt_file                       : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_URL.html */
 extern const curlopt_url                        : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_PORT.html */
 extern const curlopt_port                       : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_PROXY.html */
 extern const curlopt_proxy                      : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_USERPWD.html */
 extern const curlopt_userpwd                    : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_PROXYUSERPWD.html */
 extern const curlopt_proxyuserpwd               : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_RANGE.html */
 extern const curlopt_range                      : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_READDATA.html */
 extern const curlopt_infile                     : c_int ;
```

Comment:
Should we point out that the constant here has a slightly different name than the constant provided by libcurl?  curlopt_file would also fall under this category.  I personally feel it would be easy to miss.